### PR TITLE
Feature/wheelchair friendly path

### PR DESCRIPTION
**Pull request by pnuts2**

### CODE REVIEW

File: Classes/model/TKSegment.swift [hunk 1]
```diff
@@ -53,6 +53,16 @@ extension TKSegment {
   
 }
 
+// MARK: - Path info
+
+extension TKSegment {
+  
+  public var canShowPathFriendliness: Bool {
+    return self.template != nil && self.template.metresFriendly != nil
```
Comment (nighthawk) [inline Classes/model/TKSegment.swift]: Simpler: `return self.template?.metresFriendly != nil`

File: TripKitUI/views/TKPathFriendlinessView.swift [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        unfriendlyBarView.widthAnchor.constraint(equalTo: widthAnchor, multiplier: CGFloat(unfriendlyRatio)),
+        unknownBarView.widthAnchor.constraint(equalTo: widthAnchor, multiplier: CGFloat(unknownRatio))
+      ]
+    
+    // Lower the priority of the width constraint because floating point arithmetic may produce
+    // ratios that don't add up to precisely 1 should one of the bar views has zero width.
+    widthConstraints.forEach { $0.priority = 999 }
+    NSLayoutConstraint.activate(widthConstraints)
+    
+    let distanceFormatter = MKDistanceFormatter()
```
Comment (nighthawk) [inline TripKitUI/views/TKPathFriendlinessView.swift]: Can you add the significant digits (two?)  there? Our data isn't precise on a metre level, so the UI shouldn't pretend it is.

Comment (pnuts2) [reply TripKitUI/views/TKPathFriendlinessView.swift]: `MKDistanceFormatter` doesn't allow specifying significant digits though. `NumberFormatter` does, but `MKDistanceFormatter` isn't a subclass. `MKDistanceFormatter` also rounds the results and chooses metre/kilometre itself (cf. `NumberFormatter`). It rounds 161m to 160m, 1463m to 1.5km (also localised)

Comment (nighthawk) [reply TripKitUI/views/TKPathFriendlinessView.swift]: Copy that. All good then.

**Pull request merged**